src/xmagics/pythonexec.hpp

```diff
@@ -27,14 +29,20 @@ namespace xcpp
             std::string cline = line;
             std::string ccell = cell;
             execute(cline, ccell);
-        }
+        };
+        static void update_python_dict_var(const char * name, int value);
+        static void update_python_dict_var_vector(const char *name, std::vector<int> &data);
+        void check_python_globals();
+        void exec_python_simple_command(const std::string code);
+        static std::string transfer_python_ints_utility();
+        static std::string transfer_python_lists_utility();
+        static bool python_check_for_initialisation();
 
     private:
 
-        void startup();
+        static void startup();
```

Comment:
We probably need to make `update_python_dict_var` non static.

Reply:
It is used it `xinterpreter.cpp` that is why it is static.

Reply:
Yes, it is used within a `static` function but I don't think the intent of this other function is to be non-static either.